source/manual/alerts/slo-rate-checks-account.html.md

```diff
@@ -0,0 +1,45 @@
+---
+owner_slack: "#govuk-account-tech"
+title: GOV.UK Account SLO error rate exceeded 1% in 10 minute window
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+---
+
+The GOV.UK Accounts team has defined Service Level Objectives (SLOs) for
+reliability of the `account-api`.
+
+You can read more about these in [GOV.UK Accounts Service Level
+Objectives](https://docs.google.com/document/d/1HVXlyy2vdy9U0O-_7Nw8_Oso7VXQeKgJ2_-bi4pKlz0/edit)
+
+## Warning - "HTTP Error on GOV.UK Accounts has exceeded an acceptable threshold across the last 10 minutes"
+
+This triggers when, within a rolling 10 minute window, over 1% of all HTTP
+status codes were deemed "bad responses", as a proprotion of all responsed in
+that period.
+
+This is intended as an early warning that, if these errors continued at a
+consistent rate, the GOV.UK Accounts team is on route to spend it's entire
+error budget of 1% across the full 4 week monitoring period.
+
+This warning automatically triggers a message to the `#govuk-accounts-tech`
+slack channel.
+
+No further action is required from GOV.UK 2ndline or any other team.
+
+## Warning - "Latency on GOV.UK Accounts has exceeded an acceptable threshold across the last 10 minutes"
+
+This triggers when, within a rolling 10 minute window, over 1% of all HTTP
+responses occur with insufficently quickly, as a proprotion of all responsed in
+that period.
```

Comment:
See change in previous section.

I think there's a word missing or an extra word in line 33: "...responses occur with insufficiently quickly..."